packages/Search/src/details/DTK_DetailsDistributedSearchTreeImpl.hpp

```diff
@@ -199,12 +192,18 @@ DistributedSearchTreeImpl<DeviceType>::sendAcrossNetwork(
 
     auto imports_host = create_layout_right_mirror_view( imports );
 
-    distributor.doPostsAndWaits(
-        Teuchos::ArrayView<typename View::const_value_type>(
-            exports_host.data(), exports_host.size() ),
-        num_packets,
-        Teuchos::ArrayView<typename View::non_const_value_type>(
-            imports_host.data(), imports_host.size() ) );
+    using NonConstValueType = typename View::non_const_value_type;
+    using ConstValueType = typename View::const_value_type;
+
+    Kokkos::View<ConstValueType *, Kokkos::HostSpace,
+                 Kokkos::MemoryTraits<Kokkos::Unmanaged>>
+        export_buffer( exports_host.data(), exports_host.size() );
```

Comment:
Is this a shallow or a deep copy?

Reply:
This is not a copy, this "casts" the argument into a rank-1 view.